### PR TITLE
Upgrade to 14.0.0

### PR DESCRIPTION
**Pull request by Mudit-G**

### CODE REVIEW

File: workers/External/spatialos_worker_packages.json [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+      "path": "../../dependencies/worker_sdk",
+      "type": "worker_sdk",
+      "packages": [
+        {
+          "name": "cpp_headers"
+        }
+      ]
+    },
     {
       "path": "../../dependencies/worker_sdk/win32/debug",
```
Comment (davedissian) [inline workers/External/spatialos_worker_packages.json]: I suggest extracting to `lib/win32/debug`, `lib/win32/release`, `lib/macos` and `lib/linux` to maintain the old directory structure.

File: dependencies/CMakeLists.txt [hunk 2]
```diff
@@ -13,92 +13,52 @@ elseif(UNIX)
   endif()
 endif()
 
+set(CPP_WORKER_SDK_HEADER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/worker_sdk/include")
```
Comment (davedissian) [inline dependencies/CMakeLists.txt]: Traditionally, this is called `<library>_INCLUDE_DIR`, so perhaps call this variable `CPP_WORKER_SDK_INCLUDE_DIR`

File: dependencies/CMakeLists.txt [hunk 3]
```diff
@@ -13,92 +13,52 @@ elseif(UNIX)
...
-  NO_DEFAULT_PATH)
-find_library(LIB_WORKER_SDK_DEBUG
-  NAMES WorkerSdk
-  PATHS "${CPP_WORKER_SDK_DIR}/debug/lib" "${CPP_WORKER_SDK_DIR}/lib"
-  NO_DEFAULT_PATH)
-find_library(LIB_PROTOBUF_DEBUG
-  NAMES protobuf libprotobuf
-  PATHS "${CPP_WORKER_SDK_DIR}/debug/lib" "${CPP_WORKER_SDK_DIR}/lib"
+  NAMES improbable_worker
+  PATHS "${CPP_WORKER_SDK_DIR}/debug" "${CPP_WORKER_SDK_DIR}"
```
Comment (davedissian) [inline dependencies/CMakeLists.txt]: May be worth renaming `CPP_WORKER_SDK_DIR` to `CPP_WORKER_SDK_LIB_DIR`

File: workers/External/spatialos_worker_packages.json (hunk 1 above)
Comment (Mudit-G) [reply workers/External/spatialos_worker_packages.json]: Changed to `win32/debug/lib` etc. to match the previous directory structure.

File: dependencies/CMakeLists.txt (hunk 2 above)
Comment (Mudit-G) [reply dependencies/CMakeLists.txt]: Changed

**Pull request merged**